### ISSUE TITLE
Don’t show warning about needing h2-h6 in section if have aria-labelledby on section element?

### ISSUE THREAD

**Issue by DimaGashko** [validator/validator#982]
A few years ago this issue was closed: https://github.com/validator/validator/issues/130

So, aria-labels don't show any warnings. But... But there's can also be "aria-labelledby" attribute.

```html
<!DOCTYPE html>
<html lang="en">
<head>
   <meta charset="UTF-8">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <title>Document</title>
</head>
<body>
   
<p id="the-label">The Label</p>
<section aria-labelledby="the-label">
   <p>Some content</p>
</section>

<section aria-label="label">
   <p>Some content</p>
</section>

</body>
</html>

```

![image](https://user-images.githubusercontent.com/20650419/84154482-d895a900-aa6f-11ea-9101-a909dc66b3dd.png)
